eox_core/api/v1/views.py

```diff
@@ -200,6 +211,93 @@ def patch(self, request, *args, **kwargs):
         return Response(serialized_user.data)
 
 
+class EdxappRemoveUser(UserQueryMixin, APIView):
```

Comment:
This view has quite a bit of business logic attached to it. What if we move that to the users backend? Similar to [`create_edxapp_user`](https://github.com/eduNEXT/eox-core/blob/6080576f28557675e41e1c3964842894063288fc/eox_core/edxapp_wrapper/backends/users_j_v1.py#L63)

Reply:
done